gfx.h

```diff
@@ -6305,7 +6305,7 @@ class GfxInternal
                 installed_index_buffer_ = bound_index_buffer_;
                 force_install_index_buffer_ = false;
             }
-            if(force_install_vertex_buffer_ || bound_vertex_buffer_.handle != installed_vertex_buffer_.handle)
+            if(kernel.vertex_stride_ > 0 && (force_install_vertex_buffer_ || bound_vertex_buffer_ != installed_vertex_buffer_))
```

Comment:
Any reason behind this change?

Reply:
Because I've removed the `kernel.vertex_stride_ > 0` check from above, to allow setting index buffer below, we should ignore this branch if there are no vertex buffers for input assembler.